### DISCUSSION TITLE
Notification source storing

### DISCUSSION THREAD

**Discussion by balazs1396** [spatie/laravel-notification-log#10] [Ideas]
I would like to store the source of the notification. 
For example, if you send a notification when new post posted in a community, i would like to know, the notification which post is belongs to.

**Comment by ceejayoz** [comment]
Use the `extra` column for this. In your notification class, something like this:

```
public function logExtra(): array
{
    return ['post_id' => $this->post->id];
}
```

https://spatie.be/docs/laravel-notification-log/v1/basic-usage/adding-extra-info-to-your-notification
